tools/lint/test/util_test.py

```diff
@@ -20,6 +20,7 @@ def test_find(self):
 
         # Sanity-check relpaths.
         self.assertGreater(len(relpaths), 1_000)
+        # TODO(eric): This now fails... Why? I messed something up in the shuffle.
```

Comment:
Do you think we should be parsing .gitignore (or another such exclude file) as well here, with a more verbose message if it fails? (Is that why it was failing for you?)
https://github.com/RobotLocomotion/drake/blob/master/tools/lint/util.py#L71

Reply:
I think this actually was fixed by the following PR:
https://github.com/RobotLocomotion/drake/pull/12424

Thanks for checking!